workflow-templates/trivy.yml

```diff
@@ -0,0 +1,46 @@
+name: trivy
+
+on:
+  pull_request:
+    branches: [ "main", "develop" ]
+  schedule:
+    - cron: '0 0 1 * *'
+  push:
+    branches: [ "main", "develop" ]
```

Comment:
Can we add master here since some repos haven't switched yet

Reply:
done

Reply:
:+1: